front_end/html/edit_exercise.html

```diff
@@ -345,9 +505,57 @@ <h2>Create exercise</h2>
         {% end %}
             <a href="#top">Back to top</a>
         </p>
+        <div class="invisible" id="save_code"></div>
 
         <!-- Code to modify page behavior at the end of loading -->
         <script>
+            function saveCode() {
```

Comment:
return special characters "<", ">", and "&" after they have been converted to HTML characters in save_code (an invisible div that stores a user's code while a user is toggling example code for reference)